backstop/tests/surface.html

```diff
@@ -0,0 +1,81 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Surface | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .demo-surface-title-bar {
+        display: flex;
+        align-items: center;
+        height: 44px;
+        border-bottom: 1px solid var(--iui-color-background-5);
+        padding: 0 12px;
+      }
+      .demo-surface-content {
+        padding: 11px 12px;
+      }
+      #demo-elevations {
```

Comment:
Can you add padding to the demo sections so box-shadow would be visible?

Reply:
Test images are better now with negative margin & positive padding.